### ISSUE TITLE
UNPREDICTABLE_GAS_LIMIT

### ISSUE THREAD

**Issue by Pandasto** [t3rn/executor-release#7]
Getting this error on my node, happens randomly - deleting everything on VPS and redoing it all again helps for a while until this happens again.

`c67d424893ece899e8149f2e0a2b8b2"],"sourceNetwork":"l1rn","lifecycle":"Bid","networkId":"l1rn","isBatch":true,"lifecycle":"Bid","message":"execution reverted: BD#18 [ See: https://links.ethers.org/v5-errors-UNPREDICTABLE_GAS_LIMIT ] (method=\"estimateGas\", transaction={\"from\":\"0x490bD6792a20FA252Aa69C7109EeA441F800b955\",\"maxPriorityFeePerGas\":{\"type\":\"BigNumber\",\"hex\":\"0x05f5e100\"},\"maxFeePerGas\":{\"type\":\"BigNumber\",\"hex\":\"0x5f5e1000\"},\"to\":\"0x05C9e2dDfBa9120565c9588fd5d4464B85E77285\",\"data\":\"0x1d775d7a00000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000300000000000000000000000000490bd6792a20fa252aa69c7109eea441f800b955000000000000000000000000000000000000000000000000000000000000001498f3b98d3f3de16fc64eb95b4ec2f10ece92f250a579239040e886ca0074ca2f50ccef5d34377c50ec45c9804f8715daf1e89431ed65cf4c7ee03a86b13036388d86c4fdfd0117e7e9808312182690ba5b0cba505b4d49bbfb6a8dbce2a5813cc87438baf416329a89cc2cce01ec981440379af8ac5e40052bed5371a99cf5a8ab0d676c46075469121ee0839fded8ae42ceef4266ec1f20ae8ff77d3a0ae029c4682910fa337729bacd97ce0ba5d83eb4fffc3d4d79765df2c51c15d608c21181289730e18fbb83...","code":"UNPREDICTABLE_GAS_LIMIT","networkId":"l1rn","err":"[object Object]...","msg":"🛑 Transmitter failed to submit tx."}`

**Comment by Pandasto** [comment]
Oh and I have tried multiple different gas limits with 'export EXECUTOR_MAX_L3_GAS_PRICE=', including 50, 100, 200, 300, 400, and even 9000 but nothing works!

**Comment by weikaide-web3** [comment]
the same issue with you. Have you fix it already?

**Comment by Pandasto** [comment]
No, and I think team is on vacation at the moment so dont expect answer any time soon! 😁

There has to be some kind of fix as most people have no problems with this.

LähetettyOutlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by weikaide-web3** [comment]
I fixed this problem. Just set a high amount of gwei. I set to 50000. Maybe there are too many nodes are running caused gas masssive increased.

**Comment by Pandasto** [comment]
> I fixed this problem. Just set a high amount of gwei. I set to 50000. Maybe there are too many nodes are running caused gas masssive increased.

Did not work :( tried several values but no dice..

**Comment by weikaide-web3** [comment]
set with very high amount

**Comment by BikramBhujel** [comment]
kindly fix gas issue, i have set the gas amt to 1000

**Comment by BikramBhujel** [comment]
> 9000 but nothing works!

Delete the Binary and re-configure it, stable version is there now, set the gas to 

export EXECUTOR_MAX_L3_GAS_PRICE=1000
